libs/community/langchain_community/chat_models/anyscale.py

```diff
@@ -164,20 +160,21 @@ def validate_environment_override(cls, values: dict) -> dict:
             values["model_name"] = DEFAULT_MODEL
 
         model_name = values["model_name"]
-
-        available_models = cls.get_available_models(
-            values["openai_api_key"],
-            values["openai_api_base"],
-        )
-
-        if model_name not in available_models:
-            raise ValueError(
-                f"Model name {model_name} not found in available models: "
-                f"{available_models}.",
+        if model_name != "test":
```

Comment:
is this conditional something we want to keep?

Reply:
added this condition for unit test. But we can remove

Reply:
removed